doc/source/concepts/concepts.rst

```diff
@@ -183,7 +183,11 @@ This ensures that the independent experiments are performed in parallel rather t
 Waiting for other electrons
 ----------------------
 
-Sometimes the user might want to wait for a task's execution before executing another task even when the output of one is not the input of another. For those cases a `wait_for()` function is there which is called on the dependent electron and takes as input the depending electrons,
+Covalent normally infers the dependencies between electrons from
+their inputs and ouputs. Sometimes the user might want to wait for a
+task's execution before executing another task even when the output of
+one is not the input of another. The `wait()` function can come in
+handy in those cases.
```

Comment:
The `wait()` function can come handy in such cases?

Reply:
Good catch, thanks.